Docs/feedback_design.md

```diff
@@ -1,29 +1,129 @@
-| adoption_type | The type of adoption | 2:solution 3:command 4:scenario |
-| content | The contents adopted by users | command name/scenario name |
-| arguments | The arguments adopted by users | arguments name |
+
+#### Custom Identification
+| Field          | Content                                                        | Value                                                                              |
+|----------------|----------------------------------------------------------------|------------------------------------------------------------------------------------|
+| MachineId      | The id of the customer's machine, each device has a unique id. | 64-bit hex char (9eb810f6158f99693851fd6f203a2b7248c81563be91bd310ca03c0c10cf08c9) |
+
+
+#### Recommendation Properties
+
+Stored at Properties["context.default.azurecli.recommendation"]
+
+| Field                       | Content                                                | Value                                                                                                                                 |
+|-----------------------------|--------------------------------------------------------|---------------------------------------------------------------------------------------------------------------------------------------|
+| request_type                | The recommendation type that the user filters          | 1:all 2:solution 3:command 4:scenario 5:search                                                                                        |
+| option                      | Options adopted by users                               | str (item value: -1:no result; 0:not adopted; a$:accept the $st command; b$:accept the $st scenario; c$: accepted the $st from search |
+| trigger_commands            | The last 1-2 commands that triggers the recommendation | command name                                                                                                                          |
+| error_info                  | Error infos when using the recommendation of Solution  | error info                                                                                                                            |
+| recommendations_list        | The list of all recommendations                        | list                                                                                                                                  |                                                       
+| recommendations_source_list | The data source of recommendation items                | list (item value: 1:knowledge base 2:caculation 3:Aladdin)                                                                            |
+| recommendations_type_list   | The type of recommendation items                       | list (item value: 2:solution 3:command 4:scenario)                                                                                    |
```

Comment:
Why is there no case for the value is `1`? Because it's `all` *(same as `request_type`)*?

Reply:
Sorry for the omission, it is true that `1` stands for `ALL` in `recommendation_type_list`, I will give additional notes to it.